src/translations/fr/translations.json

```diff
@@ -240,6 +240,22 @@
       "body": "Vous recevrez le résultat du paiement à l’adresse {{useremail}}.",
       "title": "Attendez de recevoir la validation du paiement"
     },
+    "25": {
+      "body": "Non è stato addebitato alcun importo.\n Se il problema persiste, prova a usare un altro metodo o gestore del pagamento.",
+      "title": "Il pagamento non è andato a buon fine"
+    },
+    "116": {
+      "body": "Non è stato addebitato alcun importo. Prima di riprovare, aggiungi fondi alla carta o usa un altro metodo di pagamento.",
+      "title": "Il saldo della tua carta è insufficiente"
+    },
+    "117": {
+      "body": "Non è stato addebitato alcun importo. Il codice (CVV o CVC) ha 3 cifre ed è riportato sul retro della carta. Se usi American Express, il codice (CID) ha 4 cifre ed è sul fronte.",
+      "title": "Il codice di sicurezza inserito è errato"
+    },
+    "121": {
+      "body": "Non è stato addebitato alcun importo. Prima di riprovare, modifica il limite di spesa o usa un altro metodo di pagamento.",
+      "title": "Hai superato il limite di spesa della tua carta"
+    },
     "3dsNotInitiated": {
```

Comment:
should `3dsNotInitiated` be deleted? I see it deleted in the Italian translations

Reply:
“3dsNotInitiated” has not been removed in Italian translation.

Reply:
sorry :-)